### PR TITLE
Readme Improvement

### LINKED ISSUE #196: Review README.md
- Issue type:Feature request 

### Short description
Skim through and update the README.md
Review its contents and make it more helpful and uptodate for onboarding of developers.
As there are right now developers seeking to contribute it provides a good use case to get the README in a shape that is actually helpful and does not include outdated content.

https://github.com/fossasia/meilix/blob/master/README.md

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
Fixed #196 
Readme includes terms and way for the development and Hacking Meilix. Test the changes to solve the issue and best practice to make a PR.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +69,8 @@ Know your OS
 
 ## Communication
 Chat: [Gitter Channel](https://gitter.im/fossasia/meilix) | [Get an Invite](http://fossasia-slack.herokuapp.com/)
+Please join our mailing list to discuss questions regarding the project: https://groups.google.com/forum/#!forum/meilix
+Scrum report for the repository will be send to the address: meilix@googlegroups.com
```
Comment (niccokunzmann) [inline README.md]: Good to add this. Why should one send a scrum report?

File: README.md [hunk 2]
```diff
@@ -74,7 +86,7 @@ We have the following branches
  * **gh-pages**
    This contains the autogenerated code of the master branch that is generated by Travis.
  * **generator**
-   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator)
+   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator). It basically fetch the latest release from Github and use [mksquashfs tool](https://github.com/fossasia/meilix/blob/generator/build1.sh) to extract, made changes and then repack it and mail it. This will take very less time to customize the ISO.
```
Comment (niccokunzmann) [inline README.md]: He she it -> s at the end of the verb "it fetches, extracts, repacks, mails". 

less is a comparism. maybe "not much" and if you havea time, you can say about x minutes.

File: README.md (hunk 1 above)
Comment (meets2tarun) [reply README.md]: As codeheat is going on and most of the new developers asked to send the scrum report of the week, so I mentioned that.

File: README.md (hunk 2 above)
Comment (meets2tarun) [reply README.md]: @xeon-zolt /
Can you help me to mention the exact time that gets decreased, please?

Comment (xeon-zolt) [reply README.md]: 10-20 minutes

File: README.md [hunk 3]
```diff
@@ -74,7 +87,7 @@ We have the following branches
...
 
 ## License
```
Comment (xeon-zolt) [inline README.md]: https://github.com/fossasia/meilix/blob/master/LICENSE.md
 a link to file can also be added

File: README.md [hunk 4]
```diff
@@ -1,20 +1,36 @@
...
 Beautiful Linux OS for Hotels and Public Event Kiosks
 * lxqt as the standard Desktop Environment
 * based on ubuntu/debian architecture
 * Philosophy: light weight, fast, customized
 * reasonable preconfigured settings for the use case
-* system lock
+* [system lock](https://github.com/fossasia/meilix-systemlock/)
 
-## Starting Development
+# Feature
```
Comment (xeon-zolt) [inline README.md]: @meets2tarun I don't think feature is the right word to use we can use scripts maybe

**Pull request merged**